lib/cocoapods-core/source.rb

```diff
@@ -262,7 +262,7 @@ def search(query)
       if query.is_a?(Dependency)
         query = query.root_name
       end
-      found = Pathname.glob(pod_path(query)).map { |path| path.basename.to_s }
+      found = Pathname.glob(pod_path(query)).map { |path| Dir[path.to_s + '/*'].empty? ? nil : path.basename.to_s }
```

Comment:
do we care if `nil` values end up in the array? Do we want to add `.compact` in the end here?